emgapi/models.py

```diff
@@ -851,7 +897,7 @@ def available(self, request, prefetch=False):
                 Prefetch('studies', queryset=Study.objects.available(request)),
                 Prefetch('metadata', queryset=SampleAnn.objects.all())
             ).defer(
-                'is_public',
+                'is_private',
```

Comment:
Why is this deferred? Isn't it a simple no-overhead boolean column now?

Reply:
Not sure TBH...